completions/ip

```diff
@@ -318,6 +318,24 @@ _ip()
             esac
             ;;
 
+        netns)
+            case $subcmd in
+                add|identify|list|list-id|monitor)
+                    # TODO
+                    ;;
+                delete|exec|pids|set)
+                    [[ $prev == $subcmd ]] && \
+                        COMPREPLY=( $( compgen -W '$( command \
+                        ls /var/run/netns 2>/dev/null )' -- "$cur" ) )
```

Comment:
One indent level missing here.

Reply:
Instead of listing /var/run/netns, it would seem better to me to use the output from `"$1" netns list`

Reply:
You are right, using `$1 netns list` is actually better since it doesn't return error when the directory is not present.